src/toil/jobStores/abstractJobStore.py

```diff
@@ -344,45 +385,49 @@ def _importFile(self, otherCls: 'AbstractJobStore', url: ParseResult, sharedFile
         :param AbstractJobStore otherCls: The concrete subclass of AbstractJobStore that supports
                reading from the given URL and getting the file size from the URL.
 
-        :param ParseResult url: The location of the file to import.
+        :param ParseResult uri: The location of the file to import.
 
-        :param str sharedFileName: Optional name to assign to the imported file within the job store
+        :param str shared_file_name: Optional name to assign to the imported file within the job store
 
-        :return The jobStoreFileID of imported file or None if sharedFileName was given
+        :return The FileID of imported file or None if sharedFileName was given
         :rtype: toil.fileStores.FileID or None
         """
-        if sharedFileName is None:
-            with self.writeFileStream() as (writable, jobStoreFileID):
-                size, executable = otherCls._readFromUrl(url, writable)
+        if shared_file_name is None:
+            with self.write_file_stream() as (writable, jobStoreFileID):
+                size, executable = otherCls._read_from_url(uri, writable)
                 return FileID(jobStoreFileID, size, executable)
         else:
-            self._requireValidSharedFileName(sharedFileName)
-            with self.writeSharedFileStream(sharedFileName) as writable:
-                otherCls._readFromUrl(url, writable)
+            self._requireValidSharedFileName(shared_file_name)
+            with self.write_shared_file_stream(shared_file_name) as writable:
+                otherCls._read_from_url(uri, writable)
                 return None
 
+    @deprecated(new_function_name='export_file')
```

Comment:
Is there a reason we sometimes pass the new name as an arg and sometimes as a kwarg? Do both work the same?